cli/tests/test_describe_command.rs

```diff
@@ -525,29 +525,75 @@ fn test_describe_author() {
     ~
     "###);
 
-    // Reset the author for the latest commit (the committer is always reset)
+    // Change the author for the latest commit (the committer is always reset)
     test_env.jj_cmd_ok(
         &repo_path,
         &[
             "describe",
-            "--config-toml",
-            r#"user.name = "Ove Ridder"
-            user.email = "ove.ridder@example.com""#,
             "--no-edit",
-            "--reset-author",
+            "--author",
+            r#""Super Seeder <super.seeder@example.com>""#,
```

Comment:
Did you intentionally include quotes in user name?

Reply:
🤦🏻‍♂️ I was fixing a script and forgot to change my mindset that I'm using sane language again.